### ISSUE TITLE
TermKit crashes on startup

### ISSUE THREAD

**Issue by theromi** [unconed/TermKit#32]
When starting TermKit it shows up for a few seconds (unuseable, still seems to load, with a white background) and then crashes. "node termkit.js" is running.

Crashreport is here: https://gist.github.com/980495

roman-macbook:node roman$ node --version
v0.4.7

**Comment by hmottestad** [comment]
Is there anything is your logs? (console messages)

**Comment by theromi** [comment]
Nope, it doesn't seem to connect to node.

**Comment by billdozr** [comment]
I have the same problem when trying to run TermKit app.

Crash report: https://gist.github.com/980547

**Comment by hmottestad** [comment]
Maybe a firewall issue?

The crash report says EXC_BAD_ACCESS on CFArrayGetCount, which is probably an array out of bounds. There is a note of a proxy. Do you have a proxy set up?

**Comment by billdozr** [comment]
@hmottestad

You are quite right. Are there specific proxy instructions for TermKit or do I just have to just insure I just have http_proxy set?

**Comment by hmottestad** [comment]
I'm not sure. Try playing around with it and see what happens. 

I'm using a proxy on my machine (glimmerblocker) and it's not having any effect. It's a simple http proxy. What proxy settings do you have?

**Comment by hmottestad** [comment]
Seems like it's a bug in webkit: https://bugs.webkit.org/show_bug.cgi?id=47450

**Comment by hmottestad** [comment]
Might be a problem with the "Automatic Proxy Configuration" in System Preferences, Network, Advanced, Proxies

**Comment by 3rd-Eden** [comment]
Relates to: https://github.com/unconed/TermKit/issues/7

**Comment by theromi** [comment]
Jup, "Automatic Proxy Configuration" is the problem, though I don't have any proxy to autodetect..

**Comment by billdozr** [comment]
Unchecking "Automatic Proxy Configuration" does fix the problem for me.
